build-cli.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
Nothing against python per se but this introduces a python dep for something that could be replaced in the build.cmd/sh at first glance.